easybuild/easyconfigs/q/QUAST/QUAST-4.6.3-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonPackage'
+
+name = 'QUAST'
+version = '4.6.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ablab/%(namelower)s'
+description = """QUAST evaluates genome assemblies by computing various metrics.
+ It works both with and without reference genomes. The tool accepts multiple
+ assemblies, thus is suitable for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ablab/%(namelower)s/archive/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
+
+checksums = ['d7f5e670563d17d683f6df057086f7b816b6a088266c6270f7114a1406aaab63']
+
+install_target = 'install_full'
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('Perl', '5.26.1'),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Java', '1.8.0_162', '', True),
+    ('Boost', '1.66.0', versionsuffix),
```

Comment:
@RvDijk Can you check whether `QUAST` really needs Boost with Python bindings?

We are trying to limit the number of dependency variants to just 1 per toolchain generation (cfr. #5970), and we already have `2018a` easyconfigs using Boost 1.66.0 without Python bindings used as a dependency...

Reply:
@boegel not using python bindings seems fine.